src/libsystemd/sd-bus/sd-bus.c

```diff
@@ -1296,6 +1300,12 @@ _public_ int sd_bus_open_system(sd_bus **ret) {
         if (r < 0)
                 return r;
 
+        if (description) {
```

Comment:
Is it not necessary to check by `if (!isempty(description))` ?

Reply:
I don't think so. If somebody passes "" as description, it's their problem.